src/shared/data/copyContent/textareaCopy.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  default: 'default',
+  disabled: 'disabled',
+  disabledMessage: 'You will never change me! MWAAAAHAHAHAHAHAHA!',
+  errorMessage: 'Write me a story, ye heathen!',
+  pirateLore: 'Pirate lore',
```

Comment:
Let's update these names to match what we have in the [`inputCopy`](https://github.com/featherweight-design/component-library/blob/development/src/shared/data/copyContent/inputCopy.ts) (e.g. `defaultLabel` versus `default`, etc.).

Something like `pirateLore` can be `errorLabel`, which is more explicit.